src/CartridgeItem_PrinterModel.php

```diff
@@ -95,7 +95,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     public static function showForCartridgeItem(CartridgeItem $item)
     {
 
-        $instID = $item->getField('id');
+        $instID = (int) $item->getField('id');
```

Comment:
There is a dedicated method for this.

```suggestion
        $instID = $item->getID();
```

The same remark apply to other similar changes.